flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/tiered/storage/file/PartitionFileIndex.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.io.network.partition.hybrid.tiered.storage.file;
+
+import org.apache.flink.runtime.io.network.partition.hybrid.tiered.netty.NettyConnectionId;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * The {@link PartitionFileIndex} represents the indexes and the regions of the spilled buffers. For
+ * each spilled data buffer, this maintains the subpartition it belongs to, the buffer index within
+ * the subpartition, the offset in file it begin with. The {@link Region} represents a series of
+ * physically continuous buffers in the file, which are from the same subpartition.
+ */
+public interface PartitionFileIndex {
```

Comment:
Which component is this interface designed to be used by? What different implementations can it possibly have?

Reply:
Removed the API of PartitionFileIndex.